baseframe/forms/fields.py

```diff
@@ -412,7 +411,6 @@ def process_formdata(self, valuelist):
             else:
                 users = self.usermodel.all(userids=userids)
         self.data = users
-        return retval
```

Comment:
Generally not safe to make API assumptions. `__init__` explicitly does not return anything. Does WTForms explicitly say this method does not return anything? If not, don't discard the return value.